CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Backtrace Cocoa Release Notes
 
+## Version 1.7.4-beta1
+- Modify CI job to run tests daily on schedule in #81
+- fixing broken symbolification link in #80
+- Bug/crash large attachment in #84
```

Comment:
Is it accurate to say this PR skips file attachments that are larger than 10MB? or Improves handling of file attachments that are larger than 10MB?

Reply:
Your first suggestion! Yep!